### PR TITLE
New package recipe for itsxpress

### PR DESCRIPTION
**Pull request by arivers**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@bioconda/core: I am submitting a recipe for a new package and I have not submitted to Bioconda before, please review this PR.

### CODE REVIEW

File: recipes/itsxpress/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+source:
+  url: https://files.pythonhosted.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - itsxpress=itsxpress.main:main
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (bgruening) [inline recipes/itsxpress/meta.yaml]: please use `pip install`

File: recipes/itsxpress/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  skip: True # [py27]
+
+requirements:
+  build:
```
Comment (bgruening) [inline recipes/itsxpress/meta.yaml]: this should be the host section and pip should be included.

File: recipes/itsxpress/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - hmmer
+    - bbmap
+    - vsearch
+
+test:
+  imports:
+    - itsxpress
+  commands:
+    - itsxpress --help
+  requires:
```
Comment (bgruening) [inline recipes/itsxpress/meta.yaml]: do you really need to use nose?

File: recipes/itsxpress/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+about:
+  home: http://github.com/usda-ars-gbru/itsxpress
+  license: 'License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication'
+  license_family: 'PUBLIC-DOMAIN'
+  license_file: ''
+  summary: 'ITSxpress: Software to rapidly trim the Internally transcribed spacer (ITS) region of FASTQ files'
+
+
+extra:
```
Comment (bgruening) [inline recipes/itsxpress/meta.yaml]: if you can please add DOI or/and bio.tools identifier

Comment (arivers) [reply recipes/itsxpress/meta.yaml]: Added biotools and zenodo doi.

File: recipes/itsxpress/meta.yaml (hunk 2 above)
Comment (arivers) [reply recipes/itsxpress/meta.yaml]: This now reads
```
requirements:
  host:
    - pip
    - python
```

File: recipes/itsxpress/meta.yaml (hunk 3 above)
Comment (arivers) [reply recipes/itsxpress/meta.yaml]: Nose was removed.

File: recipes/itsxpress/meta.yaml (hunk 1 above)
Comment (arivers) [reply recipes/itsxpress/meta.yaml]: I now use pip. Looking at other recipes this seemed to be the most common way:
python -m pip install --no-deps --ignore-installed .
If that's not correct please let me know.

File: recipes/itsxpress/meta.yaml (hunk 3 above)
Comment (arivers) [reply recipes/itsxpress/meta.yaml]: Nose was removed, thanks.

File: recipes/itsxpress/meta.yaml (hunk 4 above)
Comment (arivers) [reply recipes/itsxpress/meta.yaml]: I have added a bio.tools entry and a Zenodo DOI.

File: recipes/itsxpress/meta.yaml (hunk 2 above)
Comment (arivers) [reply recipes/itsxpress/meta.yaml]: I've replaced build with host and added pip:
```
requirements:
  host:
    - pip
    - python
```

**Comment by arivers** [comment]
Hi @bgruening. Now that I have made the changes requested is a second review required? Do you need to merge the changes or may I do so?

**Comment by bgruening** [comment]
Can you merge master into your branch? You can merge it if it gets green.

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**